flink-doris-connector/src/main/java/org/apache/doris/flink/cfg/DorisOptions.java

```diff
@@ -107,11 +130,13 @@ public Builder setJdbcUrl(String jdbcUrl) {
         }
 
         public DorisOptions build() {
-            checkNotNull(fenodes, "No fenodes supplied.");
```

Comment:
`fenodes` should be required, because fe-related APIs, such as /schema, will be requested

Reply:
Of course